dataverse-webapp/pom.xml

```diff
@@ -679,6 +679,28 @@
                     <sourceMapOutputPath>${basedir}/src/main/webapp/resources/vecler/</sourceMapOutputPath>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>io.github.git-commit-id</groupId>
+                <artifactId>git-commit-id-maven-plugin</artifactId>
+                <version>4.9.9</version>
+                <executions>
+                    <execution>
+                        <id>get-the-git-infos</id>
+                        <goals>
+                            <goal>revision</goal>
+                        </goals>
+                        <phase>initialize</phase>
+                    </execution>
+                </executions>
+                <configuration>
+                    <generateGitPropertiesFile>false</generateGitPropertiesFile>
+                    <includeOnlyProperties>
+                        <includeOnlyProperty>^git.build.(time|version)$</includeOnlyProperty>
+                        <includeOnlyProperty>^git.commit.id.(abbrev|full)$</includeOnlyProperty>
```

Comment:
Why do we prioritize time over version?

Reply:
Why do you mean prioritize? It just means that build.time and build.version will be available as a property in .properties.

Reply:
Afaik it can be switched around and the result will be the same